spec/system/fast_edit_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+describe "Fast edit", type: :system, js: true do
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+  let(:fast_editor) { PageObjects::Components::FastEditor.new }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+  fab!(:post_2) { Fabricate(:post, topic: topic, raw: "It 'twas a great time!") }
+  fab!(:current_user) { Fabricate(:admin) }
+
+  before do
+    current_user.activate
+    sign_in(current_user)
+  end
+
+  context "when text selected it opens contact menu and fast editor" do
+    it "opens context menu and fast edit dialog" do
+      topic_page.visit_topic(topic)
+
+      topic_page.select_text("#post_1 .cooked > p", 10)
```

Comment:
I think for all these cases, `#post_X .cooked` should suffice, I don't _think_ you need the `p`. Maybe I am wrong though.

Reply:
Also better to do `topic_page.select_text("#{topic_page.post_by_number_selector(1)} .cooked", 10)`